ui/pages/confirmations/components/confirm/info/shared/redesigned-gas-fees/redesigned-gas-fees.tsx

```diff
@@ -0,0 +1,104 @@
+import { TransactionMeta } from '@metamask/transaction-controller';
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import { EditGasModes } from '../../../../../../../../shared/constants/gas';
+import {
+  addHexes,
+  multiplyHexes,
+} from '../../../../../../../../shared/modules/conversion.utils';
+import { ConfirmInfoSection } from '../../../../../../../components/app/confirm/info/row/section';
+import { currentConfirmationSelector } from '../../../../../../../selectors';
+import EditGasPopover from '../../../../edit-gas-popover';
+import { useEIP1559TxFees } from '../../hooks/useEIP1559TxFees';
+import { useSupportsEIP1559 } from '../../hooks/useSupportsEIP1559';
+import { FeesDetails } from './fees-details';
+
+function getGasEstimate(
```

Comment:
Can we call this `getGasFeeEstimate` so we're being strict with the terminology to differentiate fees and gas limits?